plaso/parsers/mobile_installation.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+"""Parser for iOS mobile installation logs files obtained from iOS device.
+Based on the research from Alex Brignoni."""
+
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class MobileInstallationEventData(events.EventData):
+  """Mobile Installation log event data
+  Attributes:
+    number (str):
+    severity (str):
+    id (str):
+    originator (str): process that created the entry
+    body (str): body of the event line
+  """
+
+  DATA_TYPE = 'mobile_installation:line'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(MobileInstallationEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.number = None
+    self.severity = None
+    self.id = None
+    self.originator = None
+    self.body = None
+
+
+class MobileInstallationParser(text_parser.PyparsingMultiLineTextParser):
+  """Parser for iOS Mobile Installation logs."""
+
+  NAME = 'mobile_installation.log'
+  DATA_FORMAT = 'iOS mobile installation logs'
+
+  MONTHS = {'Jan': 1,
+            'Feb': 2,
+            'Mar': 3,
+            'Apr': 4,
+            'May': 5,
+            'Jun': 6,
+            'Jul': 7,
+            'Aug': 8,
+            'Sep': 9,
+            'Oct': 10,
+            'Nov': 11,
+            'Dec': 12}
+
+  ONE_OR_TWO_DIGITS = text_parser.PyparsingConstants.ONE_OR_TWO_DIGITS
+  FOUR_DIGITS = text_parser.PyparsingConstants.FOUR_DIGITS
+  THREE_LETTERS = text_parser.PyparsingConstants.THREE_LETTERS
+  TIME_ELEMENTS = text_parser.PyparsingConstants.TIME_ELEMENTS
+
+  _TIMESTAMP = (THREE_LETTERS.suppress() + THREE_LETTERS('month') +
+                ONE_OR_TWO_DIGITS('day') + TIME_ELEMENTS + FOUR_DIGITS('year'))
+
+  _ELEMENT_NUMBER = pyparsing.Suppress('[') + \
```

Comment:
style nit: no need for the extra \ character here and in the other pyparsing expressions below.